synapse/cmds/hive.py

```diff
@@ -140,16 +140,16 @@ async def _handle_get(self, core, opts):
             return
 
         if opts.json:
-            prend = json.dumps(valu, indent=4, sort_keys=True)
-            rend = prend.encode()
+            rend = s_json.dumps(valu, indent=True, sort_keys=True)
+            prend = rend.decode()
         elif isinstance(valu, str):
             rend = valu.encode()
             prend = valu
         elif isinstance(valu, bytes):
             rend = valu
             prend = pprint.pformat(valu)
         else:
-            rend = json.dumps(valu, indent=4, sort_keys=True).encode()
+            rend = s_json.dumps(valu, indent=True, sort_keys=True).decode()
```

Comment:
```suggestion
            rend = s_json.dumps(valu, indent=True, sort_keys=True)
```

Reply:
`rend` is supposed to be bytes. `prend` is a string. This is correct.

Reply:
This is currently decoding the bytes into a string though?

Reply:
yep, sorry. I was confusing this with the block a few lines above. will fix.